Neos.Neos/Classes/Fusion/Helper/CachingHelper.php

```diff
@@ -76,27 +72,34 @@ public function nodeTag(iterable|Node $nodes): array
      */
     public function entryIdentifierForNode(Node $node): NodeCacheEntryIdentifier
     {
-        // Todo adjust content caching to work with workspaces as entry identifier than the content stream id
-        $currentContentStreamId = $this->workspaceNameToContentStreamIdMapping[$node->contentRepositoryId->value . '@' . $node->workspaceName->value]
-            ??= $this->contentRepositoryRegistry->get($node->contentRepositoryId)->getContentGraph($node->workspaceName)->getContentStreamId();
-
-        return NodeCacheEntryIdentifier::fromNode($node, $currentContentStreamId);
+        return NodeCacheEntryIdentifier::fromNode($node);
     }
 
     /**
      * Generate a `@cache` entry tag for a single node identifier.
      *
      * @param string $identifier
      * @param Node $contextNode
-     * @return string
+     * @return string[]
      */
-    public function nodeTagForIdentifier(string $identifier, Node $contextNode): string
+    public function nodeTagForIdentifier(string $identifier, Node $contextNode): array
     {
-        return CacheTag::forNodeAggregate(
-            $contextNode->contentRepositoryId,
-            $contextNode->workspaceName,
-            NodeAggregateId::fromString($identifier)
-        )->value;
+        return [
+            CacheTag::forNodeAggregate(
+                $contextNode->contentRepositoryId,
+                $contextNode->workspaceName,
+                NodeAggregateId::fromString($identifier)
+            )->value,
+            CacheTag::forNodeAggregate(
+                $contextNode->contentRepositoryId,
+                CacheTagWorkspaceName::ANY,
+                NodeAggregateId::fromString($identifier)
+            )->value,
+            CacheTag::forWorkspaceName(
+                $contextNode->contentRepositoryId,
+                $contextNode->workspaceName
+            )->value
+        ];
```

Comment:
I have to change the return type here, to be able to generate also the workspace tags for identifiers, which is breaking if someone relies on a string. But it works without any changes in the @cache.entryTags